website/v3/docs/features/persisted-operations.mdx

```diff
@@ -101,3 +110,30 @@ usePersistedOperations({
 ```
 
 Use this option with caution!
+
+## Using Relay's Persisted Queries Specification
+
+If you are using [Relay's Persisted Queries specification](https://relay.dev/docs/guides/persisted-queries/#example-implemetation-of-relaylocalpersistingjs), you can configure the plugin like below;
+
+```ts
+import { createYoga } from 'graphql-yoga'
+import { createServer } from 'node:http'
+import { usePersistedOperations } from '@graphql-yoga/plugin-persisted-operations'
+import persistedOperations from './persistedOperations.json'
+
+const yoga = createYoga({
+  plugins: [
+    usePersistedOperations({
+      getPersistedOperationKey(params: GraphQLParams & { doc_id: string }) {
```

Comment:
To me `extractPersistedOperationId` seems like a better fit.

`Key` confused me at first as I expected it to first return the property name within the GraphQLParams that contains the id. 😮‍💨 

I can see that key makes sense when you think of the persisted operations "store" as a key value lookup map 🤔

Reply:
Done!